src/ServiceControlInstaller.Engine/Instances/ServiceControlAuditInstance.cs

```diff
@@ -62,17 +63,13 @@ public override void Reload()
             AuditQueue = AppConfig.Read(AuditInstanceSettingsList.AuditQueue, "audit");
             AuditLogQueue = AppConfig.Read(AuditInstanceSettingsList.AuditLogQueue, $"{AuditQueue}.log");
             ForwardAuditMessages = AppConfig.Read(AuditInstanceSettingsList.ForwardAuditMessages, false);
+            AuditRetentionPeriod = TimeSpan.Parse(AppConfig.Read(AuditInstanceSettingsList.AuditRetentionPeriod, "30.00:00:00"));
             InMaintenanceMode = AppConfig.Read(AuditInstanceSettingsList.MaintenanceMode, false);
             ServiceControlQueueAddress = AppConfig.Read<string>(AuditInstanceSettingsList.ServiceControlQueueAddress, null);
             TransportPackage = DetermineTransportPackage();
             ConnectionString = ReadConnectionString();
             Description = GetDescription();
             ServiceAccount = Service.Account;
-
-            if (TimeSpan.TryParse(AppConfig.Read(AuditInstanceSettingsList.AuditRetentionPeriod, (string)null), out var auditRetentionPeriod))
```

Comment:
Given that an audit instance always has to have a retention period I removed the TryParse